src/libraries/Microsoft.PowerFx.Core/Public/Config/SymbolTable.cs

```diff
@@ -235,7 +235,7 @@ internal void AddUserDefinedFunction(string script, CultureInfo parseCulture = n
 
             errors.AddRange(parseResult.Errors ?? Enumerable.Empty<TexlError>());
 
-            if (errors.Any())
+            if (errors.Any(error => error.Severity > DocumentErrorSeverity.Warning))

```

Comment:
thanks for fixing this , can you also fix here - https://github.com/microsoft/Power-Fx/blob/f77304a685f385dc4dcb7fac58c40d4435271bee/src/libraries/Microsoft.PowerFx.Interpreter/RecalcEngine.cs#L449